pages/agent/v3/gcloud.md

```diff
@@ -2,6 +2,12 @@
 
 The Buildkite Agent can be run on [Google Cloud Platform](https://cloud.google.com). For fine control over long–lived agents, you might like to run the agent using individual VM instances on Google Compute Engine. Or run Docker–based builds using a scalable cluster of agents on the Google Kubernetes Engine using Kubernetes.
 
+## Using the Buildkite Agent Stack for Kubernetes on the Google Cloud Platform
+
+The Buildkite Agent's jobs can be run within a Kubernetes cluster on Google Cloud Platform.
```

Comment:
```suggestion
The Buildkite Agent's jobs can be run within a Kubernetes cluster on GCP.
```

Minor, just thinking about consistency here - should we say `AWS` and `GCP` or `Amazon Web Services` and `Google Cloud Platform` or `AWS (Amazon Web Services)` and `GCP (Google Cloud Platform)`? I don't have a strong preference either way but thinking we should go all initialisms or all spelled out (or both!).

Reply:
Good point, this phenomenon manifests itself right throughout this page. The usual tech writing standard is to spell out the acronym in full the first time it's used on a page (or major section that's likely to be independently linked to on the same page), mention the acronym in brackets, and then just use the acronym throughout the rest of the page or page section.

Reply:
Actually, all I needed to do was just mention the acronym in brackets at the top of the page. The way this page currently stands, this appears to be all that's required.